src/cli/cli.cc

```diff
@@ -1246,6 +1263,27 @@ int main (const int argc, const char* argv[]) {
           settings["android_allow_cleartext"] = "";
         }
       }
+
+      if (android_enable_standard_ndk_build) {
+        settings["default_config_external_native_build"].assign(
+          "    externalNativeBuild {\n"
+          "      ndkBuild {\n"
+          "        arguments \"NDK_APPLICATION_MK:=src/main/jni/Application.mk\"\n"
+          "      }\n"
+          "    }"
+        );
+
+        settings["android_external_native_build"].assign(
+          "  externalNativeBuild {\n"
+          "    ndkBuild {\n"
+          "      path \"src/main/jni/Android.mk\"\n"
+          "    }\n"
+          "  }"
+        );
+      } else {
+        settings["default_config_external_native_build"].assign("    // externalNativeBuild called manually for -j parallel support. Disable with [android]...enable_standard_ndk_build = true in socket.ini\n");
```

Comment:
oops, this should be `android_default_config_external_native_build`

Reply:
Ok, so the reason the other one was android_android is because it's related to the android settings, but it actually refers to the Android section of the gradle doc.

This one is fixed.